### PR TITLE
Update docs publish path to start with /docs

### PR DESCRIPTION
**Pull request by tomduncalf**
Required since the docs were migrated

## What, How & Why?

This closes # ???

## ☑️ ToDos

* [ ] 📝 Changelog entry
* [ ] 📝 `Compatibility` label is updated or copied from previous entry
* [ ] 🚦 Tests
* [ ] 📱 Check the React Native/other sample apps work if necessary
* [ ] 📝 Public documentation PR created or is not necessary
* [ ] 💥 `Breaking` label has been applied or is not necessary

*If this PR adds or changes public API's:*
* [ ] typescript definitions file is updated
* [ ] jsdoc files updated
* [ ] Chrome debug API is updated if API is available on React Native

### CODE REVIEW

File: .github/workflows/jsdoc-publish.yml [hunk 1]
```diff
@@ -47,6 +47,6 @@ jobs:
       # --no-mime-magic --guess-mime-type flags to work around CSS incorrect MIME type issue
       # (https://stackoverflow.com/questions/53708938/s3cmd-flagging-css-with-wrong-mime-type)
       - name: Upload versioned docs (for archival)
-        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/realm-sdks/js/${{ steps.package-version.outputs.current-version}}/
+        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/realm-sdks/docs/js/${{ steps.package-version.outputs.current-version}}/
```
Comment (nirinchev) [inline .github/workflows/jsdoc-publish.yml:50]:
```suggestion
        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/docs/realm-sdks/js/${{ steps.package-version.outputs.current-version}}/
```

File: .github/workflows/jsdoc-publish.yml [hunk 2]
```diff
@@ -47,6 +47,6 @@ jobs:
...
       - name: Upload latest docs (to live site)
-        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/realm-sdks/js/latest/
+        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/realm-sdks/docs/js/latest/
```
Comment (nirinchev) [inline .github/workflows/jsdoc-publish.yml:52]:
```suggestion
        run: s3cmd put --recursive --acl-public --no-mime-magic --guess-mime-type docs/output/realm/${{ steps.package-version.outputs.current-version}}/ s3://${{ secrets.DOCS_S3_BUCKET_NAME }}/docs/realm-sdks/js/latest/
```

**Review by nirinchev — CHANGES_REQUESTED** [review]
If I understood the change correctly, then the `docs` segment should be added between the bucket name and `realm-sdks`.

File: .github/workflows/jsdoc-publish.yml (hunk 2 above)
Comment (tomduncalf) [reply .github/workflows/jsdoc-publish.yml:52]: Ohhh thanks for spotting that, I got confused by the second mention of `realm-sdks` in the path!

**Pull request merged**